commit 61a166adea60c8f5500f7d2e300f84807b9b17ac
Author: Daniel Baumann <mail@daniel-baumann.ch>
Date:   Mon Nov 10 19:48:53 2014 +0100

    Applying patch from Gabriele Gristina <gabriele.gristina@gmail.com> to fix automated checks for using aptitude.

--- a/scripts/build/bootstrap_archives
+++ b/scripts/build/bootstrap_archives
@@ -249,8 +249,8 @@ then
 	done
 fi
 
-# Installing aptitude (FIXME)
-if [ "${LB_APT}" = "aptitude" ] && [ ! -x /usr/bin/aptitude ]
+# Installing aptitude
+if [ "${LB_APT}" = "aptitude" ] && [ ! -x chroot/usr/bin/aptitude ]
 then
 	Chroot chroot "apt-get ${APT_OPTIONS} update"
 	Chroot chroot "apt-get ${APT_OPTIONS} install aptitude"
--- a/scripts/build/chroot_archives
+++ b/scripts/build/chroot_archives
@@ -461,8 +461,8 @@ EOF
 				done
 			fi
 
-			# Installing aptitude (FIXME)
-			if [ "${LB_APT}" = "aptitude" ] && [ ! -x /usr/bin/aptitude ]
+			# Installing aptitude
+			if [ "${LB_APT}" = "aptitude" ] && [ ! -x chroot/usr/bin/aptitude ]
 			then
 				Chroot chroot "apt-get ${APT_OPTIONS} update"
 				Chroot chroot "apt-get ${APT_OPTIONS} install aptitude"
